coredhcp.go

```diff
@@ -160,7 +160,14 @@ func (s *Server) MainHandler6(conn net.PacketConn, peer net.Addr, req dhcpv6.DHC
 		resp = tmp
 	}
 
-	if _, err := conn.WriteTo(resp.ToBytes(), peer); err != nil {
+	var newPeer = peer
```

Comment:
I think this will break non relay replies since we are nos specifying a response port for those.

Reply:
Disregard